assets/js/base/components/cart-checkout/product-details/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { decodeEntities } from '@wordpress/html-entities';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+// Component to display cart item data and variations.
+const ProductDetails = ( { details = [] } ) => {
+	if ( details.length === 0 ) {
+		return null;
+	}
+
+	return (
+		<ul className="wc-block-components-product-details">
+			{ details.map( ( detail ) => {
+				return (
+					<li key={ detail.name }>
```

Comment:
Would it make sense to have a class here that would be generated using the name? This would allow a fine-grained ( for each detail independently ) styling feature.

Reply:
Good idea, added in bc9fdb738a9a584fbca14aca673911ab6b60892c.